uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/CMSVariableServiceFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use CRM_HRCore_CMSData_Variable_VariableServiceInterface as VariableAdapterInterface;
```

Comment:
If this is not the Adapter pattern, as you say in the PR description, why use the  "Adpter" term here?

Reply:
My mistake, I started with adapters and then preferred this approach. As you can see I renamed the class, but the alias wasn't updated.